templates/projects.json

```diff
@@ -1,10 +1,67 @@
-{
-  "_id": "{{@ system @}}-{{@ datatype @}}-collect",
-  "namespaced_identifiers": false,
-  "source": {
-    "operation": "{{@ datatype @}}-list",
-    "system": "{{@ system @}}",
-    "type": "rest"
+[
+  {
+    "_id": "{{@ system @}}-{{@ datatype @}}-collect",
+    "exclude_completeness": [
+      "{{@ system @}}-{{@ datatype @}}-share"
+    ],
+    "namespaced_identifiers": false,
+    "source": {
+      "operation": "{{@ datatype @}}-list",
+      "supports_since": false,
+      "system": "{{@ system @}}",
+      "trace": {},
+      "type": "rest"
+    },
+    "transform": [
+      {
+        "properties": {
+          "primary_key": "id",
+          "rest_system": "{{@ system @}}",
+          "share_dataset": "{{@ system @}}-{{@ datatype @}}-share"
+        },
+        "template": "transform-collect-rest",
+        "type": "template"
+      },
+      {
+        "rules": {
```

Comment:
Why does this transform exist?

Reply:
just old crud left over from a failed attempt to calculate a "$last-modified" value. I've removed it now.